src/ripple/RippleEngine.ts

```diff
@@ -30,6 +30,7 @@ import {
 import {
   getBalanceChanges,
   OfferCreate,
+  Payment as PaymentJson,
```

Comment:
I'll trust you that upstream `Payment` is a subtype of the removed `PaymentJson`